crypto/zkp/primitives/commitments/damgard-fujisaki_positive_test.go

```diff
@@ -0,0 +1,69 @@
+/*
+ * Copyright 2017 XLAB d.o.o.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+package commitmentzkp
+
+import (
+	"math/big"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/xlab-si/emmy/crypto/commitments"
+	"github.com/xlab-si/emmy/crypto/common"
+)
+
+// TestProveDamgardFujisakiCommitmentPositive demonstrates how to prove that the commitment
+// hides a positive number. Given c, prove that c = g^x * h^r (mod n) where x >= 0.
+func TestProveDamgardFujisakiCommitmentPositive(t *testing.T) {
```

Comment:
Please update all error messages in this file so that they start with a lowercase letter.